force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -160,9 +191,11 @@ public with sharing class VariablesDataTypesOperators {
      * @return The number of milliseconds equivalent to the given number of minutes.
      */
     public static Integer convertMinutesToMilliseconds(Integer minutes) {
-        Integer MILLISECONDS_PER_MINUTE = null; // Make this value a constant
+        Integer MILLISECONDS_PER_MINUTE = minutes * 60000; // Make this value a constant
```

Comment:
Try to make this a constant by using the final keyword, you can also remove it from the method and put it outside by using the static final keywords.